packages/core/src/impl-store.ts

```diff
@@ -36,6 +43,7 @@ async function fetchOrDeployGeneric<T extends Deployment, U extends T = T>(
   deploy: () => Promise<U>,
   opts?: DeployOpts,
   merge?: boolean,
+  getDeploymentResponse?: (deploymentId: string, allowUndefined: boolean) => Promise<DeploymentResponse | undefined>,
```

Comment:
This new param is missing documentation. Same for the other functions.

Reply:
I realize now that the entire function is missing documentation. :stuck_out_tongue: That's on me. Could be a good moment to add some though.

Reply:
I realize now that the entire function is missing documentation. :stuck_out_tongue: That's on me. Could be a good moment to add some though.